torch/fx/experimental/symbolic_shapes.py

```diff
@@ -407,10 +428,10 @@ def create_symbolic_sizes_strides(self, ex: torch.Tensor):
                 )[0]
                 stride[i] = self.create_symbol(val)
         assert all(x is not None for x in stride)
-        return [self.create_symintnode(i) for i in size], [self.create_symintnode(i) for i in stride]  # type: ignore[arg-type]
+        return [self.create_symintnode(x, id(ex), "size", idx) for idx, x in enumerate(size)], [self.create_symintnode(x, id(ex), "stride", idx) for idx, x in enumerate(stride)]  # type: ignore[arg-type] # noqa: B950
 
-    def create_symintnode(self, expr: Union["sympy.Expr", int]):
-        py_sym_int = PySymInt(expr, self)
+    def create_symintnode(self, expr: Union["sympy.Expr", int], ref_id: Optional[int] = None, kind: Optional[str] = None, idx: Optional[int] = None):  # noqa: B950
```

Comment:
the noqa here is unnecessary, you can put the arguments on a new line...

or if you don't want to spend time reformatting, black the file in a prep PR before this one.